src/lib.rs

```diff
@@ -0,0 +1,609 @@
+use std::collections::HashMap;
+
+use borsh::schema::{BorshSchemaContainer, Declaration, Definition, Fields, VariantName};
+use schemars::schema::{RootSchema, Schema};
+use serde::{Deserialize, Serialize};
+
+#[doc(hidden)]
+#[cfg(feature = "__chunked-entries")]
+#[path = "private.rs"]
+pub mod __private;
+
+/// Current version of the ABI schema format.
+pub const SCHEMA_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+/// Contract ABI.
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq)]
+pub struct AbiRoot {
+    /// Semver of the ABI schema format.
+    abi_schema_version: String,
```

Comment:
Why is this not public? If someone doesn't use the chunked-entries feature with that constructor, there is no way for them to construct this. Intentional? If you just wanted to rename, could you not just use `#[serde(alias = "abi_schema_version")]`?

Reply:
Hm, Is there a use case for manually constructing an `AbiRoot`? I'm thinking it's either deserialized from contract inspection or it's generated through `cargo-near`.

Reply:
I think this repo should contain non-biased Rust models for the general public: anyone can take these models and build on top of them however they like. So yeah I feel like all models should be freely constructible.

Reply:
yeah, not a blocking comment, just thinking of the re-usability of this library outside of our specific use case